src/onnx/parse_squeeze.cpp

```diff
@@ -0,0 +1,62 @@
+#include <migraphx/onnx/op_parser.hpp>
+#include <migraphx/ranges.hpp>
+#include <migraphx/make_op.hpp>
+#include <migraphx/onnx/checks.hpp>
+#include <migraphx/instruction.hpp>
+
+namespace migraphx {
+inline namespace MIGRAPHX_INLINE_NS {
+namespace onnx {
+
+struct parse_squeeze : op_parser<parse_squeeze>
+{
+    std::vector<op_desc> operators() const
+    {
+        return {{"Squeeze", "squeeze"}, {"Unsqueeze", "unsqueeze"}};
+    }
+
+    bool needs_contiguous(const std::string& op_name) const
+    {
+        return contains({"squeeze", "unsqueeze"}, op_name);
+    }
+
+    void assign_axes(const operation& op, const std::vector<int64_t>& axes) const
```

Comment:
This function looks like a nop. What is it supposed to be doing?

Reply:
I forgot a return value. Fixed.